src/common/api.js

```diff
@@ -91,7 +91,7 @@ export const getForgeManifest = () => {
 };
 
 export const getFabricManifest = () => {
-  const url = `${FABRIC_APIS}/v2/versions?timestamp=${new Date().getTime()}`;
+  const url = `${FABRIC_APIS}/versions?timestamp=${new Date().getTime()}`;
```

Comment:
What about removing the timestamp, there is no need for this. it just puts more load on the meta api, it already has a short cache time of 60 seconds.

Reply:
There weren't no timestamps (on the three manifests, including this one) until [this commit](https://github.com/gorilla-devs/GDLauncher/commit/89eb7c4e66024bc4bbd88ea0e05d6a5de219ce26), it looks like that without them, the manifests were cached.

Reply:
They should be cached, I see no reason for them not to be cached.

Reply:
Alright, I'm removing the timestamp from the request for Fabric's manifest. I'm leaving the other 2 for someone else, I don't want to break anything silently.